### ISSUE TITLE
4.4.1 Unexpected "missing whitespace" errors in end of file

### ISSUE THREAD

**Issue by justtal** [palantir/tslint#2129]
### Bug Report

- __TSLint version__: `4.4.1`
- __TypeScript version__: `2.1.5`
- __Running TSLint via__: `CLI`

#### TypeScript code being linted

```ts
any .ts file
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "quotemark": [
      true,
      "single",
      "avoid-escape"
    ],
    "class-name": true,
    "no-consecutive-blank-lines": true,
    "semicolon": [
      true,
      "always"
    ],
    "whitespace": [
      true,
      "check-branch",
      "check-decl",
      "check-operator",
      "check-module",
      "check-separator",
      "check-type"
    ],
    "no-var-keyword": true,
    "interface-name": [true, "always-prefix"],
    "adjacent-overload-signatures": true,
    "use-isnan": true,
    "one-variable-per-declaration": true
  }
}
```

#### Actual behavior

Unexpected "missing whitespace" errors in all files (in the last char of the file)

#### Expected behavior

No unexpected errors

Thanks!

**Comment by nchen63** [comment]
I can't reproduce. can you put it in a zip file and attach it? Also, what OS?

**Comment by justtal** [comment]
OS X Yosemite.

[mock.enum.ts.zip](https://github.com/palantir/tslint/files/731682/mock.enum.ts.zip)

**Comment by larshp** [comment]
I have the same issue, see https://github.com/larshp/abaplint/pull/134, also it does not happen in 4.3.1 for me

it looks like it is related to the `check-separator` option
